src/graph/impl/KokkosGraph_Distance2MIS_impl.hpp

```diff
@@ -405,14 +405,15 @@ struct D2_MIS_RandomPriority
     int decideSetTeamSize = 0;
     if(useTeams)
     {
+      team_pol temp(1, 1, vectorLength);
```

Comment:
may need to find a better name than temp ?

Reply:
@srajama1 I just renamed it to ``dummyPolicy``.